src/templates/remote_crates.bzl.template

```diff
@@ -6,11 +6,20 @@ DO NOT EDIT! Replaced on runs of cargo-raze
 
 def {{workspace.gen_workspace_prefix}}_fetch_remote_crates():
 {% for crate in crates %}
+    {%- if crate.git_data %}
+    native.new_git_repository(
```

Comment:
Nit: Should we default to clone submodules as well: `init_submodules = true`?

Reply:
I left it off just to keep with the default behavior for `new_git_repository`. Definitely not opposed to putting it in there, and I'm not sure that we could possibly break anything by doing so.

Now that I'm thinking about it, I'll go ahead and add that by default, we can revisit in the future if it turns out to be an issue, though I have no idea under what circumstances that could be the case (since Cargo has already figured out where the manifest directory is).